symfony/src/EventListener/RequestListener.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\EventListener;
+
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use App\Service\LoggerService;
+
+class RequestListener
+{
+    private $logger;
+
+    public function __construct(LoggerService $logger)
+    {
+        $this->logger = $logger;
+    }
+
+    public function onKernelRequest(RequestEvent $event)
+    {
+        if (!$event->isMasterRequest()) {
```

Comment:
Even after reading [the docs](https://symfony.com/doc/current/event_dispatcher.html) on this, I'm still not sure about sub requests.